spec/rmagick/image/read_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe Magick::Image, '#read' do
   describe 'issue #200' do
     before do
-      pid = Process.spawn File.join(SUPPORT_DIR, 'issue_200', 'app.rb'), err: :close, out: :close
+      pid = Process.spawn "ruby #{File.join(SUPPORT_DIR, 'issue_200', 'app.rb')}", err: :close, out: :close
```

Comment:
Looks like at least one failure is caused by this test. I think [we can just comment out the entire `before` block](https://github.com/rmagick/rmagick/pull/342), since both tests below are `skip`ped anyway.